service/src/main/java/greencity/service/notification/NotificationServiceImpl.java

```diff
@@ -787,6 +787,16 @@ public void deleteNotification(Long notificationId, String userUuid) {
         userNotificationRepository.markAsDeletedUserNotificationByIdAndUserId(notificationId, userId);
     }
 
+    @Override
+    public void notifyUnpaidOrderPermanently(Order order, Long amountToPay) {
+        boolean isOrderPayed = order.getOrderPaymentStatus().equals(OrderPaymentStatus.PAID);
+        if (!isOrderPayed) {
+            Double amount = amountToPay.doubleValue() / 100.0;
```

Comment:
why do we need this? 
please move `100.0` to the constant

Reply:
A good point, I've already made it a constant